### PR TITLE
ansible: install logrotate package on nginx role

### PR DESCRIPTION
**Pull request by santigimeno**

### CODE REVIEW

File: ansible/roles/nginx/tasks/main.yml [hunk 1]
```diff
@@ -7,7 +7,10 @@
 #
 
 - name: install
-  package: name=nginx state=present
+  package: name={{ item }} state=present
```
Comment (jbergstroem) [inline ansible/roles/nginx/tasks/main.yml]: quote please

File: ansible/roles/nginx/tasks/main.yml [hunk 2]
```diff
@@ -7,7 +7,10 @@
 #
 
 - name: install
```
Comment (jbergstroem) [inline ansible/roles/nginx/tasks/main.yml]: nitpick: seeing how we're going from only installing nginx to installing multiple packages perhaps change to "install required packages"

File: ansible/roles/nginx/tasks/main.yml (hunk 1 above)
Comment (santigimeno) [inline ansible/roles/nginx/tasks/main.yml]: Changing it but, is it necessary?

**Comment by santigimeno** [comment]
Updated. PTAL. Thanks!

**Pull request merged**